templates/data/test-infra/pjtester-data.yaml

```diff
@@ -25,13 +23,22 @@
                 - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/vpathguard.sh"
             jobConfig_pjtester:
               command: "pjtester"
+              env:
+                KUBECONFIG_PATH: /etc/kubeconfig/pjtester/config
+              labels:
+                preset-pjtester-kubeconfig: "true"
```

Comment:
You're adding here a preset that you're deleting from `prow/config.yaml` at the same time